subliminal/video.py

```diff
@@ -147,6 +163,18 @@ def __repr__(self):
             return '<%s [%r]>' % (self.__class__.__name__, self.title)
         return '<%s [%r, %r]>' % (self.__class__.__name__, self.title, self.year)
 
+    def __hash__(self):
+        if self.year is None:
+            return hash((
+                self.title,
+                self.year,
+            ))
+        return hash(self.title)
```

Comment:
I'm OK to keep it as it makes total sense for library users, I didn't see direct use of this in the library so I just wondered why add this :)